.github/workflows/migration-tests.yml

```diff
@@ -209,6 +209,9 @@ jobs:
           AZURE_ATLAS_APP_ID: ${{vars.AZURE_ATLAS_APP_ID}}
           AZURE_SERVICE_PRINCIPAL_ID: ${{vars.AZURE_SERVICE_PRINCIPAL_ID}}
           AZURE_TENANT_ID: ${{vars.AZURE_TENANT_ID}}
+          AWS_REGION: ${{ vars.AWS_REGION_LOWERCASE }}
```

Comment:
why does it have to be lowercase specifically in this case?

Reply:
because it's for AWS to use it to login in using STS.   uppercase is for Atlas.  (the strange thing for me is to have the AWS_REGION var in uppercase)